modules/mobile/app/policies/schedule_appointment_policy.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+ScheduleAppointmentPolicy = Struct.new(:user, :schedule_appointment) do
+  def access?
+    accessible = user.loa3? && user.vha_facility_ids.any?
```

Comment:
Does `user.vha_facility_ids.any?` differ meaningfully from the `user.va_patient?` method Patrick recommended? That method seems to be a bit more involved but the comments around the method seem to imply that it's essentially the same as what you have here.

Reply:
I didn't use `va_patients?` because it gets overridden in `app/models/iam_user.rb` to a method that doesn't actually correctly determine va patient status (according to comment above it.). Per your suggestion over slack, I'll just use the `va_treatment_facility_ids` to ensure non-valid facility ids are still getting filtered out.